File: src/main/java/ai/aimachineserver/api/HomeController.java
package ai.aimachineserver.api;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
class HomeController {
    @GetMapping
    String homeMessage() {
        return "Greetings from AIMachine Backend service";
    }
}

File: src/main/kotlin/ai/aimachineserver/users/HomeController.kt
package ai.aimachineserver.users

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController("/")
class HomeController {
    @GetMapping
    fun homeMessage(): String {
        return "Greetings from AIMachine Backend service"
    }
}
